bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/discovery/TeslaVehicleDiscoveryService.java

```diff
@@ -81,31 +81,15 @@ public void deactivate() {
 
     @Override
     public void vehicleFound(Vehicle vehicle, VehicleConfig vehicleConfig) {
```

Comment:
Please add `@Nullable` to the parameter `vehicleConfig`.

Reply:
The whole class/binding does not yet have null annotations on it, I wouldn't want to do it as part of a bug fix.